From 7d2374237c682b8d68833db1272d710e12d46157 Fri, 16 Dec 2016 13:15:14 +0100
From: Islam Amer <islam.amer@jollamobile.com>
Date: Wed, 12 Nov 2014 18:10:53 +0200
Subject: Tracker config overrides

Set the default delay for GraphUpdated to 350 instead of upstream default of 1000. Helps fixing JB#11570

diff --git a/data/org.freedesktop.Tracker.Store.gschema.xml.in b/src/tracker-store/org.freedesktop.Tracker.Store.gschema.xml.in
index f7c4565..9c29b0e 100644
--- a/data/org.freedesktop.Tracker.Store.gschema.xml
+++ b/data/org.freedesktop.Tracker.Store.gschema.xml
@@ -25,7 +25,7 @@
       <description>Log verbosity.</description>
     </key>
     <key name="graphupdated-delay" type="i">
-      <default>1000</default>
+      <default>350</default>
       <summary>GraphUpdated delay</summary>
       <description>Period in milliseconds between GraphUpdated signals being emitted when indexed data has changed inside the database.</description>
     </key>

